softaware.Authentication.Hmac.AspNetCore/HmacAuthenticationHandler.cs

```diff
@@ -178,6 +178,13 @@ private bool IsReplayRequest(string nonce, string requestTimeStamp)
             return false;
         }
 
+        private string GetRequestScheme(HttpRequest req)
+        {
+            if (this.Options.TrustProxy && req.Headers.TryGetValue("X-Forwarded-Proto", out var scheme))
```

Comment:
Please conform to our coding guidlines and add square brackets for the if-statement. Thank you.

Reply:
done